### ISSUE TITLE
training data format

### ISSUE THREAD

**Issue by kminooie** [facebookresearch/fastText#1118]
Could we have a formal definition of what the structure of each line of data should be some where in the documentation? The only thing that I could find in the docs was the actual file that where being used the tutorial. My specific question is whether I can have space in my labels and if so how do I specify where my last label ends and my data begins. Thanks.

**Comment by candacelax** [comment]
The data should be a list of "__label__LABEL TEXT" lines. So for example:
__label__excited i just got a new job and i can't wait
__label__sad my dog just died
...

**Comment by kminooie** [comment]
so, no space or white character of any kind in the label?

**Comment by candacelax** [comment]
Yep, no whitespace in the label! Then your text can just be whitespace formatted (or tokenized with something like NLTK)
